View/Helper/BootstrapModalHelper.php

```diff
@@ -60,7 +62,14 @@ public function create($title = null, $options = array()) {
             $this->currentId = $options['id'] ;
             $options['aria-labbeledby'] = $this->currentId.'Label' ;
         }
-		$res = $this->Html->div('modal fade', NULL, $options).$this->Html->div('modal-dialog').$this->Html->div('modal-content');
+        if (isset($options['size'])) {
+            if($options['size'] == 'large'){$size = ' modal-lg';} 
+            if($options['size'] == 'small'){$size = ' modal-sm';} 
+        } else {
+            $size = null; //default size modal
+        }
+
```

Comment:
You should `unset($options['size'])`, because if you don't one one the `div` will have `size` attribute.